release: v1.56.0 (#6071)
apollographql/router: .changesets/fix_latte_lens_tent_blaze.md

> ### Internal `apollo_private.*` attributes are not sent to the Jaeger collector ([PR #6033](https://github.com/apollographql/router/pull/6033))
> 
> When using Jaeger collector to send traces you will no longer receive span attributes with the `apollo_private.` prefix which is a reserved internal keyword not intended to be externalized.

Reviewer: ```suggestion
### Prevent sending internal `apollo_private.*` attributes to Jaeger collector ([PR #6033](https://github.com/apollographql/router/pull/6033))

When using the router's Jaeger collector to send traces, you will no longer receive span attributes with the `apollo_private.` prefix. Those attributes were incorrectly sent, as that prefix is reserved for internal attributes.
```
